.travis.yml

```diff
@@ -48,6 +48,10 @@ install:
     - cd ..
     # we need cstor code
     - git clone https://github.com/openebs/cstor.git
+    - cd cstor
+    - if [ ${TRAVIS_BRANCH} == "master" ]; then git checkout develop; else git checkout ${TRAVIS_BRANCH} || git checkout develop; fi
+    - git branch
+    - cd ..
```

Comment:
should we remove line 64 of new file i.e., git checkout develop

Reply:
Yes. Thanks for pointing out.